jujugui/static/gui/src/app/init.js

```diff
@@ -35,6 +35,11 @@ const yui = window.yui;
 class GUIApp {
   constructor(config) {
     /**
+      Stash the location object so that tests can override it.
+      @type {Object}
+    */
+    this.location = window.location;
```

Comment:
Is this necessary? Do we actually use this in the app anywhere or was it only added to appease a test? I'm pretty sure that is an artifact of the old Y.App routing.

Reply:
You're right, nothing was failing before I added that, except for the test that checks it exists.